packages/webapp/pages/squads/discover/my.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react';
```

Comment:
Right. If that's the case, we should probably have introduced all pages from `/discover` to the new `app` folder. However, it would increase the scope as the Next14 upgrade was unexpected and not planned here. I think we should do it on the next ones.

Reply:
Ticket: https://dailydotdev.atlassian.net/jira/software/projects/MI/boards/31/backlog?epics=visible&selectedIssue=MI-527